flixel/tweens/FlxTween.hx

```diff
@@ -1120,14 +1170,120 @@ class FlxTweenManager extends FlxBasic
 		_tweens.splice(0, _tweens.length);
 	}
 
+	/**
+	 * Cancels all related tweens on the specified object.
+	 *
+	 * Note: Any tweens with the specified fields are cancelled, if the tween has other properties they
+	 * will also be cancelled.
+	 * 
+	 * @param Object The object with tweens to cancel.
+	 * @param FieldPaths Optional list of the tween field paths to search for. If empty, all tweens on the specified
+	 * object are canceled. Allows dot paths to check child properties.
+	 */
+	public function cancelTweensOf(Object:Dynamic, ?FieldPaths:Array<String>):Void
+	{
+		forEachTweensOf(Object, FieldPaths, function (tween) tween.cancel());
+	}
+
+	/**
+	 * Immediately updates all tweens on the specified object with the specified fields that
+	 * are not looping (type `FlxTween.LOOPING` or `FlxTween.PINGPONG`) and `active` through
+	 * their endings, triggering their `onComplete` callbacks.
+	 *
+	 * Note: if they haven't yet begun, this will first trigger their `onStart` callback.
+	 *
+	 * Note: their `onComplete` callbacks are triggered in the next frame.
+	 * To trigger them immediately, call `FlxTween.globalManager.update(0);` after this function.
+	 *
+	 * In no case should it trigger an `onUpdate` callback.
+	 *
+	 * Note: Any tweens with the specified fields are completed, if the tween has other properties they
+	 * will also be completed.
+	 *
+	 * @param Object The object with tweens to complete.
+	 * @param FieldPaths Optional list of the tween field paths to search for. If empty, all tweens on
```

Comment:
If empty, or if null? Or both? A bit strange that the doc comment only talks about the empty case when the field is optional.

Reply:
Gonna change all of these to say "If null or empty" and will change `forEachTweensOf` to check for empty